ember-flatpickr/rollup.config.mjs

```diff
@@ -30,6 +30,7 @@ export default {
       'helpers/**/*.js',
       'modifiers/**/*.js',
       'services/**/*.js',
+      'initializers/**/*.js',
```

Comment:
```suggestion
```

Reply:
I kept it as a souvenir of my work ... ahah
But yeah let's remove this line 😄